app/models/manageiq/providers/openstack/cloud_manager/provision_workflow.rb

```diff
@@ -2,6 +2,7 @@ class ManageIQ::Providers::Openstack::CloudManager::ProvisionWorkflow < ::MiqPro
   def allowed_instance_types(_options = {})
     source                  = load_ar_obj(get_source_vm)
     flavors                 = get_targets_for_ems(source, :cloud_filter, Flavor, 'flavors')
+    return flavors if flavors.blank?
```

Comment:
should this return something more predictable like `nil` or empty hash or empty array?